components/search/SearchResultItem.tsx

```diff
@@ -12,21 +12,34 @@ type SearchResultItemProps = {
 const SearchResultItem = ({ searchResult }: SearchResultItemProps) => {
   const image = useImage(imageSizes.poster.w154, searchResult.posterPath);
 
+  // mx-2 my-1 md:mx-2.5 md:my-1.5
```

Comment:
What are these margin values for?

Reply:
These are old margin values for the text of the item. But i need to rework this somewhere in the future.